pkg/odo/cli/catalog/search/service.go

```diff
@@ -29,30 +32,74 @@ func NewSearchServiceOptions() *SearchServiceOptions {
 
 // Complete completes SearchServiceOptions after they've been created
 func (o *SearchServiceOptions) Complete(name string, cmd *cobra.Command, args []string) (err error) {
-	o.Context = genericclioptions.NewContext(cmd)
-	o.searchTerm = args[0]
+	if experimental.IsExperimentalModeEnabled() {
+		var noCsvs, noServices bool
+		o.Context = genericclioptions.NewContext(cmd)
```

Comment:
In case of experimental mode being enabled, we use `NewDevfileContext`.

Reply:
done

Reply:
@dharmit can't do so...
if you check below, actually 

it calls non-experiment code
```
o.services, err = catalog.SearchService(o.Client, o.searchTerm)
```
so this has to be NewContext